src/ImageSharp.Web.Providers.AWS/AmazonS3ClientFactory.cs

```diff
@@ -9,6 +9,8 @@ namespace SixLabors.ImageSharp.Web;
 
 internal static class AmazonS3ClientFactory
 {
+    static AmazonS3ClientFactory() => AWSConfigs.InitializeCollections = true;
```

Comment:
Altering global behavior for transitive dependency does not appear be a prudent. It would be more appropriate to update the code to accommodate changes in behavior.